### PR TITLE
Create navigation menu items

### LINKED ISSUE #960: Create menu items for Shop/Cart/Checkout
The menu doesn’t have cart/checkout/shop pages

image

### Possible solution

It seems like there is a post in the posts table that holds the navigation

post type: wp_navigation
post_title: primary

We could check if those shop/cart/checkout pages exist, and if not append something like 

```
<!-- wp:navigation-link {"label":"Checkout","type":"page","id":773,"url":"https://devpanos.wpcomstaging.com/checkout/","kind":"post-type","isTopLevelLink":true} /-->

<!-- wp:navigation-link {"label":"Shop","type":"page","id":769,"url":"https://devpanos.wpcomstaging.com/shop/","kind":"post-type","isTopLevelLink":true} /-->
```

I did a quick test with PHPMyAdmin and it works - BUT I need to  `wp cache flush`

If there is a better (non hacky way) would be great

cc @daledupreez @xristos3490

### PR DESCRIPTION
**Pull request by PanosSynetos**
### Changes proposed in this Pull Request:

Closes #960 .

### How to test the changes in this Pull Request:

- [x] Rsync `add/960-ecomm-menu-items` on an existing atomic ecommerce site
- [x] SSH into the site,  edit `includes/class-wc-calypso-bridge-setup.php`
and comment out the following, in `woocommerce_create_pages_callback`

```
			if ( WCAdminHelper::is_wc_admin_active_for( 300 ) ) {
				update_option( $this->option_prefix . $operation, 'completed', 'no' );

				return;
			}

```

This will make the one-time operation to run, even the site is not new.

- [x] Using wp cli (hint if proxied visit https://testsite.xxxx.com/_cli) run `wp option delete wc_calypso_bridge_one_time_operation_woocommerce_create_pages`
- [x] Run `wp option get wc_calypso_bridge_one_time_operation_woocommerce_create_pages` and see that the job is completed
- [x] Navigate to the frontend of your site, and see that the navigation menu is Shop, Cart, Checkout, My Account.

### CODE REVIEW

File: includes/class-wc-calypso-bridge-setup.php [hunk 1]
```diff
@@ -299,6 +334,7 @@ public function woocommerce_create_pages_callback() {
 				// Update and Release row.
 				$wpdb->query( 'COMMIT' );
 				wp_cache_delete( $this->option_prefix . $operation, 'options' );
+				wp_cache_flush();
```
Comment (xristos3490) [inline includes/class-wc-calypso-bridge-setup.php:337]: Why do we need this? Does the menu has cache on the output?

File: includes/class-wc-calypso-bridge-setup.php [hunk 2]
```diff
@@ -286,6 +286,41 @@ public function woocommerce_create_pages_callback() {
...
+					);
+
+					$menu_content = '<!-- wp:navigation-link {"label":"' . __( 'Home', 'woocommerce' ) . '","url":"/","kind":"custom","isTopLevelLink":true} /-->';
+
+					foreach ( $menu_pages as $key => $page ) {
+						if ( ! is_a( $page, 'WP_Post' ) ) {
+							continue;
+						}
+
+						$title = $page->post_title;
```
Comment (xristos3490) [inline includes/class-wc-calypso-bridge-setup.php:307]: Is this safe for special chars? Like single and double quotes?

File: includes/class-wc-calypso-bridge-setup.php (hunk 1 above)
Comment (PanosSynetos) [reply includes/class-wc-calypso-bridge-setup.php]: Removed

File: includes/class-wc-calypso-bridge-setup.php (hunk 2 above)
Comment (PanosSynetos) [reply includes/class-wc-calypso-bridge-setup.php:307]: Updated

**Review by xristos3490 — APPROVED** [review]
Looks good!

**Pull request merged**